.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,21 @@
 ## Summary
+<!-- A short, clear statement of the proposed changes. -->
 
-Summarize your PR. If it involves visual changes include a screenshot or gif.
+## How to test
```

Comment:
Open to removing this, but IMO it's kind of a vital part of a PR that's missing at the moment. I don't think we need to be overly prescriptive in how this is written, just that it is there in _some_ way.